library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-                  ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-EMG-features {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Other-polygraphic-channel {requireChild}
+        ** Myoclonus <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Negative-myoclonus <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-rhythmic <nowiki> [Numerical value for frequency. (Source: Beniczky ea 2017, Table 16.)] </nowiki>
+                *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+            *** Myoclonus-arrhythmic <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-synchronous <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+            *** Myoclonus-asynchronous <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** PLMS <nowiki>[Periodic limb movements in sleep. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+        ** Spasm <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** Tonic-contraction <nowiki> [Source: Beniczky ea 2017, Table 16.] </nowiki>
+        ** Other-EMG-features <nowiki> {requireChild} [Source: Beniczky ea 2017, Table 16.]</nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+    * Other-polygraphic-channel <nowiki> {requireChild}[Source: Beniczky ea 2017, Section 13.] </nowiki>
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
 
 '''Finding-property''' <nowiki>[Descriptive element similar to main HED /Property. Something that pertains to a thing. A characteristic of some entity. A quality or feature regarded as a characteristic or inherent part of someone or something. HED attributes are adjectives or adverbs.]</nowiki>
 
-    * Signal-morphology-property <nowiki>[Signal morphology pertains to the visual and occasionally structural attributes employed for the description and characterization of findings.]</nowiki>
+    * Signal-morphology-property <nowiki>[Signal morphology contains attributes to describe findings in background, interictal or ictal activity. (Source: Beniczky ea 2017, Table 5, Table 8, Table 12.)]</nowiki>
 
-        ** Rhythmic-activity-morphology <nowiki>[EEG activity consisting of a sequence of waves approximately constant period.]</nowiki>
-            *** Delta-activity-morphology <nowiki> {suggestedTag=Finding-frequency, suggestedTag=Finding-amplitude} [EEG rhythm in the delta (under 4 Hz) range that does not belong to the posterior dominant rhythm (scored under other organized rhythms).]</nowiki>
+        ** Rhythmic-activity-attribute <nowiki>[Rhythmic activity can be observed during background, interictal or ictal activity and HED-SCORE therefore describes this as an property/attribute. (Source: Beniczky ea 2017, Table 5, Table 12, Table 14.)]</nowiki>
```

Comment:
Maybe  Rhythmic-activity-attribute  -->  Rhythmic-activity

Reply:
Not sure, we have Rhythmic-activity-pattern (line 160, normal), Abnormal-interictal-rhythmic-activity (line 110, interictal) and the Rhythmic-activity-attribute can provide attributes for both the normal and interictal patterns.

Reply:
To align across HED, should this be `Rhythmic-activity-property` instead? @VisLab

Reply:
Will be changes to rhythmic-property

Reply:
fixed